backend/app/configs/docker/kafka/bitnami_kafka.yaml

```
@@ -26,3 +28,8 @@ services:
 
 volumes:
   kafka1_data: null
+
+networks:
+  kafka_default:
+    driver: bridge
+    external: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)